ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -1730,7 +1713,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh";
+			shellScript = "export SENTRY_PROPERTIES=sentry.properties\nexport NODE_BINARY=node\n../node_modules/@sentry/cli/bin/sentry-cli react-native xcode ../node_modules/react-native/scripts/react-native-xcode.sh";
```

Comment:
Something tells me that we need another \n in here, or we'll wind up running "sentry-cli React-native" which I don't think is a command

Reply:
This was autogenerated, and I don't think it should be a problem.

Reply:
I'm game - if CI builds, it's not a problem here